src/main/java/de/aservo/atlassian/confapi/rest/api/SettingsResource.java

```diff
@@ -8,12 +8,8 @@
 import io.swagger.v3.oas.annotations.responses.ApiResponse;
 
 import javax.validation.constraints.NotNull;
-import javax.ws.rs.Consumes;
 import javax.ws.rs.GET;
 import javax.ws.rs.PUT;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.core.MediaType;
```

Comment:
vielleicht macht es ja sogar Sinn, diese annotations hier auch am Interface zu haben. dann erzwingen wir Einheitlichkeit über alle apps hinweg. nicht dass im einen Fall json geliefert wird und in der anderen app versehentich xml....

Reply:
Das müssten wir nochmal ausprobieren, ob `@Produces` funktioniert.

`@Path` geht leider nicht, weil er dann versucht, das Interface zu instanziieren...

Reply:
schade...